src/sqlancer/mysql/MySQLOptions.java

```diff
@@ -24,6 +25,9 @@ public class MySQLOptions implements DBMSSpecificOptions<MySQLOracleFactory> {
     @Parameter(names = "--oracle")
     public List<MySQLOracleFactory> oracles = Arrays.asList(MySQLOracleFactory.TLP_WHERE);
 
+    @Parameter(names = "--execute-queries", description = "Specifies whether the query in the fuzzer should be executed", arity = 1)
```

Comment:
Do we actually need this option? Users can set https://github.com/sqlancer/sqlancer/blob/7804a3adec0962ad6d24687c42ec473aa49669fe/src/sqlancer/MainOptions.java#L38 to 0 so that no oracle is executed.

Reply:
Removing this option from SQLite, MySQL, PostgreSQL.